gtk4/src/auto/bool_filter.rs

```diff
@@ -25,7 +25,7 @@ glib::wrapper! {
 
 impl BoolFilter {
     #[doc(alias = "gtk_bool_filter_get_expression")]
-    pub fn get_expression(&self) -> Expression {
+    pub fn get_expression(&self) -> Option<Expression> {
```

Comment:
This is not nullable

Reply:
but maybe it should be marked as such?

Reply:
It's not nullable yes, I just fixed the gir warning here and want to tackle the rest of #68 or at least parts of it soon :-) I would leave this for a separate PR

Reply:
OTOH you can create a bool filter with a `NULL` expression so... what would this return then?

Reply:
Hmm, indeed. The set_expression would fail if you pass it a `NULL` for the expression, it shouldn't work when you initialize it neither. Opened an upstream issue for now https://gitlab.gnome.org/GNOME/gtk/-/issues/3779

Reply:
I guess the answer is in the issue now. Can you update the PR accordingly?

Reply:
I.e. the expression must be either `NULL` or its value type must be `bool`.

Reply:
I added checks to both BoolFilter & StringFilter :-)